pkg/metrics/backup.go

```diff
@@ -43,6 +43,7 @@ func NewBackupMetrics() BackupMetrics {
 				"files_skipped_bytes", "cluster", "manifest", "keyspace", "table"),
 			filesFailedBytes: gr("Number of bytes failed to download from backup location (local to current restore run).",
 				"files_failed_bytes", "cluster", "manifest", "keyspace", "table"),
+			batchSize: gr("Size of the files batch taken by host.", "batch_size", "cluster", "host"),
```

Comment:
But this metric is strictly related to the batch itself and it suppose to be used to monitor how the batches are distributed between nodes.
I asked ChatGPT about correctness of the sentence :)))

> The sentence you have provided is grammatically correct, but it may not be entirely clear without additional context. It appears to be describing the size of a batch of files taken by a host, but it is unclear what the purpose or significance of this batch is.

After some conversation with this lovely tool we ended up with 
```
Cumulative size of the batches of files taken by the host to restore the data
```

AI answered with 
> The sentence is grammatically correct and conveys a clear message. It informs the reader that the host has taken batches of files for the purpose of restoring the data, and asks for the total size of all these batches.

I'm bit of afraid that "Number of bytes assigned to be restored by host." lacks the context of batch.

Changing it to **Cumulative size of the batches of files taken by the host to restore the data**